src/registry.json

```diff
@@ -1451,6 +1451,29 @@
         }
       }
     },
+    "npm:xdcdomains-snap": {
+      "id": "npm:xdcdomains-snap",
+      "metadata": {
+        "name": "XDCWeb3Domains",
+        "author": {
+          "name": "XDCDomains",
+          "website": "https://xdcdomains.xyz/"
+        },
+        "summary": "Easily transfer assets using .xdc domain names instead of complex wallet addresses.",
```

Comment:
```suggestion
        "website": "https://xdcdomains.xyz/metamask/",
        "summary": "Easily transfer assets using .xdc domain names instead of complex wallet addresses.",
```